beeai/openshift/deployment-backport-agent.yml

```diff
@@ -0,0 +1,79 @@
+apiVersion: apps/v1
+kind: Deployment
+spec:
+  progressDeadlineSeconds: 600
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app: backport-agent
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: backport-agent
+        deployment: backport-agent
+    spec:
+      containers:
+      - args:
+        - agents/backport_agent.py
+        command:
+        - /usr/bin/python
+        env:
+        - name: MCP_JIRA_URL
+          valueFrom:
+            configMapKeyRef:
+              key: MCP_JIRA_URL
+              name: x-beeai-agent-configmap
+        - name: REDIS_URL
+          valueFrom:
+            configMapKeyRef:
+              key: REDIS_URL
+              name: x-beeai-agent-configmap
+        - name: COLLECTOR_ENDPOINT
+          valueFrom:
+            configMapKeyRef:
+              key: COLLECTOR_ENDPOINT
+              name: x-beeai-agent-configmap
+        - name: MAX_RETRIES
+          valueFrom:
+            configMapKeyRef:
+              key: MAX_RETRIES
+              name: x-beeai-agent-configmap
+        - name: GITLAB_TOKEN
+          valueFrom:
+            secretKeyRef:
+              key: GITLAB_TOKEN
+              name: beeai-agent-secrets
+        - name: GITLAB_USER
+          valueFrom:
+            secretKeyRef:
+              key: GITLAB_USER
+              name: beeai-agent-secrets
+        - name: CHAT_MODEL
+          valueFrom:
+            secretKeyRef:
+              key: CHAT_MODEL
+              name: beeai-agent-secrets
+        - name: GEMINI_API_KEY
+          valueFrom:
+            secretKeyRef:
+              key: GEMINI_API_KEY
+              name: beeai-agent-secrets
+        - name: DRY_RUN
+          value: "true"
+        image: 'quay.io/jotnar/beeai:latest'
+        imagePullPolicy: Always
+        name: backport-agent
+        ports:
+        - containerPort: 8080
+          protocol: TCP
+        resources: {}
+        terminationMessagePath: /dev/termination-log
+        terminationMessagePolicy: File
+      dnsPolicy: ClusterFirst
+      restartPolicy: Always
+      schedulerName: default-scheduler
+      securityContext: {}
+      terminationGracePeriodSeconds: 30
```

Comment:
I'm not sure if that's enough 👀

Reply:
What shall be added here?

Reply:
Would a TODO comment be sufficient here?